DEVELOPMENT.md

```diff
@@ -25,30 +25,30 @@ I'd recommend following this process to iterate:
 keep code changes in a branch.
 1. Run `mvn clean install` or the Maven tool window of the IDE to build this
 connector, while addressing errors/problems from build output.
-1. Config a separate prestosql server with the built connector by creating a file
+1. Config a separate trinodb server with the built connector by creating a file
 named `docker-compose.yml`:
     ```YAML
     # docker-compose.yml
     version: "3.7"
 
     services:
-    presto-coordinator:
-        image: prestosql/presto:347
-        container_name: presto-coordinator
+    trino-coordinator:
+        image: trinodb/trino:<trino_verson_from_pom>
+        container_name: trino-coordinator
         volumes:
-        - source: ./target/presto-db2-347
-        target: /usr/lib/presto/plugin/db2
+        - source: ./target/trino-db2-<trino_verson_from_pom>
```

Comment:
since it still points to `presto-db2` in `Dockerfile`  
https://github.com/IBM/trino-db2/blob/112b4cc9ea63aaa935131ddab0a41ed2674498f1/Dockerfile#L3

Should the source and target be `presto-db2` ?

Reply:
Notice that nothing really seems to even use the Dockerfile. The image that is being used in the suggested `docker-compose.yaml` is based on the upstream image on dockerhub. I think the Dockerfile is kind of an artifact at this point.